tests/domain/display/test_display.py

```diff
@@ -20,28 +22,63 @@
                 dist_km=0.1,
                 speed_kph=2.4,
             ),
-            "Distance: 0.10 km. Duration: 1m. Speed: 2.4 km/h.",
-            f"{chr(27)}[2J{chr(27)}[HDistance: 0.10 km\nDuration: 1m\nSpeed: 2.4 km/h",
-            '{"distance_m": 100, "duration_s": 60, "speed_kph": 2.4}',
+            DailySummary(),
```

Comment:
should test more variants:
* 0 values (currently tested)
* non-zero values
* None